### ISSUE TITLE
Accounts created by this script are being suspended

### ISSUE THREAD

**Issue by igamer1337** [silvestrodecaro/microsoft-account-creator#3]
The majority of accounts generated using this script are experiencing suspensions, necessitating phone verifications to unlock.

**Comment by silvestrodecaro** [comment]
this happens because you are creating very poor quality accounts, the causes can be the following:
1. Bad IP/ Bad Proxy;
2. Browser not updated;
3. Bad Browser (if selected);
4. Bad Browser Profile (if selected).

**Comment by igamer1337** [comment]
You're doing remarkable work here, and I truly appreciate it. However, I've noticed an issue in the script. After creating the account, the generative window closes immediately, preventing users, particularly Email client software users like myself from signing in directly. Attempts to add the account result in a "bad email and password" error. To overcome this, I've found a workaround by first logging in to Outlook on the browser, receiving the welcome email, and then adding the account to email client software. It would be great if this could be addressed for a smoother user experience.
